com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -1,9 +1,12 @@
+//#define PROFILE_BUILD
```

Comment:
Do the profiling calls actually cause that much overhead? Sounds like the markers are in a too tight loop if that is the case. If they're causing that noticeable overhead, they probably won't actually give useful profiling data.

Reply:
Actually this line should be removed as the way to set these defines should be done on the project settings.
 
This is guarded because I enable Profiler to write a file using `Profiler` API. I actually don't know if there's an overhead to have the Profiler enabled but there's a huge cost to disk writing (one of my files had 40GB on disk).